src/app/SanctionedCountryErrorScreen.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react'
+import { StyleSheet, Text, View } from 'react-native'
+import fontStyles from 'src/styles/fonts'
+
+class SanctionedCountryErrorScreen extends React.Component {
+  render() {
+    return (
+      <View style={styles.container}>
+        <Text style={styles.errorMessage}>{'Sorry, Valora is not support in your location'}</Text>
```

Comment:
hum, should we localise this? the chances of the app supporting the user's device language is pretty low though. i'm not sure what the right answer is 😅 i guess we can leave it like this, there is a typo here though. it should be `Sorry, Valora is not supported in your location`

Reply:
Oh... I just copied and pasted the sentence in the linear ticket lol. I changed it!
I think it would not be a good idea to localize this because we don't need to support localization for those who can't use our app.